nikola/conf.py.in

```diff
@@ -250,6 +250,16 @@ HIDDEN_TAGS = ['mathjax']
 # CATEGORY_PATH = "categories"
 # CATEGORY_PREFIX = "cat_"
 
+# If CATEGORY_ALLOW_HIERARCHIES is set to True, categories can be organized in
+# hierarchies. For a post, the whole path in the hierarchy must be specified,
+# using a forward slash ('/') to separate paths. Use a backslash ('\') to escape
+# a forward slash or a backslash (i.e. '\//\\' is a path specifying the
```

Comment:
Maybe someone likes to have Unix paths in her/his categories, like `/dev/null`. Having that as a subcategory would be impossible if we parse left-to-right. I really don't like the idea of not being able to specify certain (sub-)category names just because we wanted to make things a little simpler.